nih-sparc/sparc-app
# Clear search results redesign

# Description
The purpose of this PR is to update the clear search results design and UX.
![localhost_3000_(Laptop with MDPI screen) (1)](https://user-images.githubusercontent.com/1493195/79012844-4d626a80-7b35-11ea-82d3-6dcc0c95509e.png)


## Ticket
[Wrike ticket](https://www.wrike.com/workspace.htm?acc=3203588#path=folder&t=467387669&a=3203588&id=473706657&st=space-441356195)

## Type of change


# How Has This Been Tested?
- Go to the Find Data page
- The X icon in the input should not be visible if a search hasn't been made
- Enter text and click on the search icon to submit a search
- The X icon should now be visible
- Click on the X icon
- The search should be reset


# Checklist:
